### PR TITLE
{Icemoon, Ruin} The Crashed Holemaker

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Adds a brand-new - well, [new to shiptest](https://cohost.org/bluwu016/post/885120-a-compilation-of-my) - ruin to the Icemoon roster, focused on the service department.

It's flavored around being an *incredibly* old NT Spaceworks vessel that's been carved in half and crashed - what's present only being the fore of the ship. Being mainly service-focused, it's loot is pretty dry ~~as is it's sole threat.~~
If more current-day mappers/balance-heads have any words about how to fluff out either of those pools a bit more with the screenshots below, lemme know. I'll listen well.

(Notarized loot summary removed as updating it was a pain in the ass, lmao.)

It strikes me as leaning on the underwhelming side from looking at the other ruins present here but we'll. See? I suppose? It's good practice for me in the whole, "making something I have memorized and that looks good normally look sicker ruined".

<details><summary>Pictures (All but SDMM Outdated)</summary>
<p>

Ignore that there's no rust, the firelocks are open here, and some stuff's knocked around, I was testing it prior to me tacking the rust on and took pics after running around it in-person.

image

image

StrongDMM View:

</p>
</details> 

## Why It's Good For The Game
This isn't what I intended to do when I was like, "oh yeah, I have a goofy ahh downstream out of boredom, ya'll want some of our better ships" but w/e here it is anyways. Ya'll need ruins. I made (another) ruin.

## Changelog

:cl:
add: A new icemoon ruin has been added, should you be in need of service department goodies.
/:cl:

### CODE REVIEW

**Review by spockye — COMMENTED** [review]
we could probably use some less loot-intensive ruins 
Not seeing any issues, I'll give it a TM Soonish!

**Review by Latentish — CHANGES_REQUESTED** [review]
Make a ruin catalogue entry.

**Comment by unit0016** [comment]
Addressed

File: config/iceruinblacklist.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@
 #_maps/RandomRuins/IceRuins/icemoon_underground_brazillianlab.dmm
 #_maps/RandomRuins/IceRuins/icemoon_underground_burnies_lair.dmm
 #_maps/RandomRuins/IceRuins/icemoon_underground_wendigo_cave.dmm
+#_maps/RandomRuins/IceRuins/icemoon_crashed_holemaker.dmm
```
Comment (Latentish) [inline config/iceruinblacklist.txt]: This is a deprecated file and is not being maintained, nor should be altered.

**Review by Latentish — DISMISSED** [review]

**Comment by Latentish** [comment]
This seems like a good stash of loot for something with no real threats.. Also I think you could benefit from adding some exterior padding, some of the environment around it, so by default it doesn't spawn out in the open.

**Comment by unit0016** [comment]
> This seems like a good stash of loot for something with no real threats.. 

I'll update this, but honestly, I *really* don't believe that it's the case.
Even ignoring that other pre-existing ruins are thousands times more egregious (As examples I can name off the top of my head from playing, see `icemoon_surface_engioutpost.dmm`'s free lathe setup and full engineering kit for three people and `jungle_surface_roommates.dmm` for two *40 force highlander claymores*), the two big ticket items are.. a bar setup (two bar dispensers, a reagent grinder, a maintenance booze-o-mat, a condimaster) - and a service protolathe\*, which is more restricted than even the autolathe, save a *few* items that can't be moved over to an autolathe via a design disk and RND.

Everything else is either redundant or otherwise incredibly minor.

\* NOT a techfab, to be clear. This can't print circuit boards. Yes, they look IDENTICAL. It annoys the hell out of me too!! I tried to fix this on /tg/ but a conflict happened and I got lazy.

> Also I think you could benefit from adding some exterior padding, some of the environment around it, so by default it doesn't spawn out in the open.

Going to be taking this advice in *partially* the opposite direction tbh. Thinking on it more I want only the south side to be exposed by default.

**Review by Erikafox — CHANGES_REQUESTED** [review]
This is very detail-sparse for a crash map. Please include scrap and debris trailing the crash site.
image
subtype this so it has proper atmos please. Active turfs bad.
image
ditto

Could use some creatures / definition stuff inside to show that it's been there a bit, and didn't experience the joys of atmospheric re-entry 20 minutes ago. Blown in snow piles, corpses, stuff like that.

**Comment by unit0016** [comment]
No updatepaths script, cringe

**Comment by unit0016** [comment]
Did all of that. In the process, the following items have been added to the ruin and thus it's.. "loot" ""pool"":
- Two slags of corgi meat and a corgi hide
- A microwave
- A fueltank and mechanical toolbox
- a KNIFE (no!!)
- WOODDD. More wood! There's now enough for a baseball bat just lying around, but it's scattered.
- A double bedsheet.
- Brand spanking new mime and assistant husk corpse spawners. Also the generic corpse spawner icon has been redone to differentiate it a bit (purely because mob/landmarks.dmi had an icon state that didn't correspond to any job)

I was considering tacking on a pacman but decided against it last minute.

File: code/modules/awaymissions/corpse.dm [hunk 2]
```diff
@@ -364,6 +367,14 @@
 	outfit = /datum/outfit/job/clown
 	icon_state = "corpseclown"
 
+/obj/effect/mob_spawn/human/mime
```
Comment (thgvr) [inline code/modules/awaymissions/corpse.dm]: This isn't necessary - we don't support clown/mime content in ruins

**Review by thgvr — CHANGES_REQUESTED** [review]

File: code/modules/awaymissions/corpse.dm (hunk 2 above)
Comment (unit0016) [reply code/modules/awaymissions/corpse.dm]: alrighty, I'll swap out the body for a ~~botanist~~ space bartender corpse (tacking on a plant analyzer is a nuh uh and I don't want to snowflake just for this) then, that's fitting enough.

Comment (unit0016) [reply code/modules/awaymissions/corpse.dm]: actually wait, does that encompass the ventriloquist dummy I already had in there considering that's part of the mime toolkit, or is that harmless?

Comment (thgvr) [reply code/modules/awaymissions/corpse.dm]: Naw that's fine. Plant analyzers are in autolathes rn anyways

**Review by thgvr — APPROVED** [review]

**Comment by Erikafox** [comment]
> Did all of that. In the process, the following items have been added to the ruin and thus it's.. "loot" ""pool"":
> 
>     * Two slags of corgi meat and a corgi hide
> 
>     * A microwave
> 
>     * A fueltank and mechanical toolbox
> 
>     * a KNIFE (no!!)
> 
>     * WOODDD. More wood! There's now enough for a baseball bat just lying around, but it's scattered.
> 
>     * A double bedsheet.
> 
>     * Brand spanking new mime and assistant husk corpse spawners. Also the generic corpse spawner icon has been redone to differentiate it a bit (purely because mob/landmarks.dmi had an icon state that didn't correspond to any job)
> 
> 
> I was considering tacking on a pacman but decided against it last minute.

Go a little more wild with the debris and assorted random bullshit. Loot Pools are fake because most of this stuff is common enough anyways. Scatter around broken machinery bits so people can reassemble them, more wood, Just. More. It still feels weirdly empty to me.
I will duely inform you if it ends up being Too Much.

**Comment by unit0016** [comment]
Hokay, that should be most of the whitespace filled up?

**Comment by unit0016** [comment]
Noticing MDB has an issue here where it shows the wrong icon state for chairs and chairs *specifically.* Weird!

**Review by Erikafox — APPROVED** [review]
Gonna see how this plays out today.

**Comment by goober3** [comment]
LGTM

**Pull request merged**